Add docs from ros2_control repo. (#5)
ros-controls/control.ros.org: index.rst

   :depth: 2

Overview
=========
The ros2_control framework consists of the following packages:

* `ros2_control`_ - core functionalities;
* `ros2_controllers`_ - general controllers' implementations;
* `control_toolbox`_ - usefull functionalities for controllers;
* `realtime_tools`_ - usefull functionalities for realtime support, e.g., realtime buffers and publishers;
* `control_msgs`_ - common messages.

Additionally, there are following (unreleased) packages are relevant for getting-started and project management:

* `ros2_control_demos`_ - example implementations of the framework;
* `roadmap`_ - planning and design docs for the project.

Development Organisation and Communication
-------------------------------------------

WG Meeting
   Every second Wednesday there is a Working Group meeting.

>    To joint the meeting check the announcement on `ROS Discourse`_.

Reviewer: ```suggestion
   To join the meeting check the announcement on `ROS Discourse`_.
```

   You can joint the meeting through `google groups <https://groups.google.com/forum/#!forum/ros-control-working-group-invites>`_ or directly on Google Meet (check the announcement).

>    To review, add disscusion points, and check older discussions and notes check `this document <https://docs.google.com/document/d/1818AoYucI2z82awL_-8sAA5pMCV_g_wXCJiM6SQmhSQ/edit?usp=sharing>`_.

Reviewer: ```suggestion
   To propose new discussion points, or review notes from previous meetings, check `this document <https://docs.google.com/document/d/1818AoYucI2z82awL_-8sAA5pMCV_g_wXCJiM6SQmhSQ/edit?usp=sharing>`_.
```

Projects
   GitHub `projects under ros-control organization <https://github.com/orgs/ros-controls/projects>`_ are used to track the work.

Bug reports and feature requests

>    Use the issue tracker in the correspoding repository for this.

Reviewer: ```suggestion
   Use the issue tracker in the corresponding repository for this.
   Give a short summary of the problem
   Make sure to provide a minimal list of steps one can follow to reproduce the issue you found
   Provide relevant information regarding the operating system, ROS distribution, etc.
```

Questions 
   Please use `ROS Answers <https://answers.ros.org/>`_ and tag your questions with ``ros2_control``.

General discussions
   Please use `ROS Discourse`_.

Getting Started
================

> The ros2_control framework is released for ROS2 foxy.

Reviewer: ```suggestion
The ros2_control framework is released for ROS2 Foxy.
```

> To use it, you have to install ``ros-foxy-ros2-control`` and ``ros-foxy-ros2-controller` packages.

Reviewer: ```suggestion
To use it, you have to install ``ros-foxy-ros2-control`` and ``ros-foxy-ros2-controllers` packages.
```
